torchci/lib/JobClassifierUtil.ts

```diff
@@ -14,12 +15,16 @@ export const groups = [
   },
   {
     regex: /unstable/,
-    name: "Unstable Jobs",
+    name: "Unstable",
```

Comment:
It's a bit ugly, but this name is used as a constant below in `isUnstableGroup` function, so you would need to change it there as well

Reply:
Thanks for catching that! I've updated that check to be a bit more flexible